challenge-6.js

```diff
@@ -0,0 +1,14 @@
+function pyramid(n = 5) {
+  for (let index = 1; index <= n; index++) {
+    for (let j = 1; j <= n - index; j++) {
+      process.stdout.write(" ");
```

Comment:
Since we are practicing to become a Front End Developer, it is better to avoid using `process` in these kinds of situations and use `console.log`